Content.Server/Body/Systems/BrainSystem.cs

```diff
@@ -33,12 +39,23 @@ private void HandleMind(EntityUid newEntity, EntityUid oldEntity)
             if (HasComp<BodyComponent>(newEntity))
                 ghostOnMove.MustBeDead = true;
 
+            if (HasComp<UnborgableComponent>(oldEntity)) // DeltaV
+                EnsureComp<UnborgableComponent>(newEntity);
+
             if (!_mindSystem.TryGetMind(oldEntity, out var mindId, out var mind))
                 return;
 
             _mindSystem.TransferTo(mindId, newEntity, mind: mind);
         }
 
+        private void OnExamined(Entity<UnborgableComponent> ent, ref ExaminedEvent args) //DeltaV
+        {
+            var msg = new FormattedMessage();
+            msg.AddMarkupPermissive("[color=red]This brain is damaged beyond use.[/color]");
```

Comment:
this should be localized and use AddMarkupOrThrow then
also not sure why you need to create a new message

Reply:
you can use pushmarkup yes